assets/src/dashboard/app/views/templateDetails/components.js

```diff
@@ -108,6 +108,10 @@ export const RowContainer = styled.section.attrs({
   flex-wrap: wrap;
   padding-bottom: 40px;
   margin: 0 20px 0;
+
+  & > div {
+    width: 100%;
+  }
```

Comment:
grid is now nested because of the useGridKeyboard functionality that wants a certain structure for ref and keydown tracking. Adding this to maintain the layout.